modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java

```diff
@@ -136,6 +136,7 @@ public NotificationTemplate addNotificationTemplate(
 		notificationTemplate.setEditorType(
```

Comment:
change commit message to `Object Actions of Notification type should only create custom notification templates drafts`